_posts/2022-08-31-quarkus-helm.adoc

```diff
@@ -0,0 +1,251 @@
+---
+layout: post
+title: 'Generate Helm charts manifests with Quarkus Helm'
+date: 2022-08-30
+tags: extension
+synopsis: How to use the Quarkus Helm extension to generate the Helm chart manifests
+author: jcarvaja
+---
+
+https://helm.sh/[Helm] is the most popular package manager that finds, shares, and deploys software built for Kubernetes. To configure Java applications for distribution through Helm, you need to package them as Helm charts containing all of the necessary metadata and resource definitions and this can be a very tedious task. The good news is that Quarkus has recently released the new https://github.com/quarkiverse/quarkus-helm[Quarkus Helm] extension that will automatically generate all these Helm chart manifests and resources!
```

Comment:
```suggestion
https://helm.sh/[Helm] is the most popular package manager that finds, shares, and deploys software built for Kubernetes. To configure Java applications for distribution through Helm, you need to package them as Helm charts containing all the necessary metadata and resource definitions and this can be a very tedious task. The good news is that Quarkus has recently released the new https://github.com/quarkiverse/quarkus-helm[Quarkus Helm] extension that will automatically generate all these Helm chart manifests and resources!
```